src/uwtools/tests/test_schemas.py

```diff
@@ -61,18 +61,18 @@ def ungrib_prop():
 
 
 def test_schema_chgres_cube():
-    d = {
+    config = {
```

Comment:
Perhaps a slightly more meaningful name.

Reply:
This is a great change.